subworkflows/nf-scil/preproc_t1/tests/nextflow_synth.config

```diff
@@ -0,0 +1,14 @@
+process {
```

Comment:
Once you've added the `params.run_synth_bet` to the `main.nf` file, use it here.